source/guide.md

```diff
@@ -0,0 +1,8 @@
+---
+layout: guide
+title: Sass Basics
+introduction: >
+  Before you can use Sass, you need to set it up on your project. If you want to
+  just browse here, go ahead, but we recommend you go install Sass first. [Go
+  here](/install) if you want to learn how to get everything set up.
+---
```

Comment:
@sanajaved7 Is there a benefit to having this file? I think we could delete this, and move `_layouts/guide.liquid` here instead (`source/guide.liquid`) and it would work the same?